src/VisualStudio/Core/Def/TableDataSource/DiagnosticTableItem.cs

```diff
@@ -43,11 +43,14 @@ public override TableItem WithAggregatedData(string[] projectNames, Guid[] proje
         public override ProjectId? ProjectId
             => Data.ProjectId;
 
+        // TODO: use of OriginalFileSpan seems very suspect here.  It is used for navigation.  But we should likely
```

Comment:
should we have a tracking issue for this.  Although I guess it becomes moot once we switch to lsp diags